packages/nextjs/package.json

```
@@ -0,0 +1,67 @@
+{
+  "name": "@fedify/nextjs",
```

I know it's very trivial and I'm picky about it, since Next.js' npm package name is just [*next*](https://www.npmjs.com/package/next), it would be better to be *@fedify/next* here:

```suggestion
  "name": "@fedify/next",
```